aws/sdk/integration-tests/webassembly/src/default_config.rs

```diff
@@ -3,26 +3,10 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-use aws_config::retry::RetryConfig;
-use aws_credential_types::Credentials;
-use aws_smithy_types::timeout::TimeoutConfig;
-use aws_types::region::Region;
 use std::future::Future;
 
-use crate::adapter::Adapter;
-
 pub(crate) fn get_default_config() -> impl Future<Output = aws_config::SdkConfig> {
-    aws_config::from_env()
-        .region(Region::from_static("us-west-2"))
-        .credentials_provider(Credentials::from_keys(
-            "access_key",
-            "secret_key",
-            Some("session_token".to_string()),
-        ))
-        .timeout_config(TimeoutConfig::disabled())
-        .retry_config(RetryConfig::disabled())
-        .http_connector(Adapter::to_http_connector())
-        .load()
+    aws_config::from_env().region("us-west-2").load()
```

Comment:
this would be something like `aws_config::from_env().http_connector(wasm_connector())` — you probably need to also set it in the provider config as well. We can make a top level setter for that to make it easier.

Reply:
a new function has been exposed to create the connector for wasi within wasm crate